crates/bevy_ecs/src/archetype.rs

```diff
@@ -112,7 +112,7 @@ impl ArchetypeId {
 #[derive(Copy, Clone, Eq, PartialEq)]
 pub(crate) enum ComponentStatus {
     Added,
-    Mutated,
+    Existing,
```

Comment:
Doc strings on this type and its enum variant would be really helpful for reviewers.